pkg/promtail/targets/tailer.go

```diff
@@ -135,3 +139,57 @@ func (t *tailer) stop() error {
 func (t *tailer) cleanup() {
 	t.positions.Remove(t.path)
 }
+
+// Fatal implements tail.logger
```

Comment:
I think we shouldn't add methods to the `tailer`. I'd rather have an adapter in `pkg/util` that does this.

Reply:
sure will make the change.